formats/json/commonMain/src/kotlinx/serialization/json/JsonElementBuilders.kt

```diff
@@ -186,6 +198,33 @@ public fun JsonArrayBuilder.addJsonObject(builderAction: JsonObjectBuilder.() ->
 public fun JsonArrayBuilder.addJsonArray(builderAction: JsonArrayBuilder.() -> Unit): Boolean =
     add(buildJsonArray(builderAction))
 
+/**
+ * Adds the given JSON [elements] to a resulting JSON array.
+ *
+ * @return `true` if the list was changed as the result of the operation.
+ */
+@ExperimentalSerializationApi
+public fun JsonArrayBuilder.addAll(vararg elements: JsonElement): Boolean {
```

Comment:
Same for `vararg` here

Reply:
Oops, I missed that one. It's now removed.

Reply:
Also `apiDump`

Reply:
hey @sandwwraith, sorry I forgot this! Would you mind updating it? You should have permissions.

Reply:
Ok, no problems